go/vt/wrangler/materializer.go

```diff
@@ -1305,3 +1312,28 @@ func (mz *materializer) forAllTargets(f func(*topo.ShardInfo) error) error {
 	wg.Wait()
 	return allErrors.AggrError(vterrors.Aggregate)
 }
+
+// checkIfTimeZoneIsLoaded is a light-weight consistency check to validate that, if a source time zone is specified to MoveTables,
+// that the current primary has the time zone loaded in order to run the convert_tz() function used by VReplication to do the
+// datetime conversions. We only check the current primaries on each shard and note here that it is possible a new primary
+// gets elected: in this case user will either see errors during vreplication or vdiff will report mismatches.
+func (mz *materializer) checkIfTimeZoneIsLoaded(ctx context.Context, tz string, keyspace string) error {
```

Comment:
nit but I think a better name is `checkTZConversion` or `validateTZConversion` since we're ensuring that the named time zone info is available and the specified source timezone is a valid name.